includes/admin_settings.php

```diff
@@ -332,7 +346,59 @@ function tsml_settings_page()
                         </form>
                     </div>
 
```

Comment:
Check the order of these on the settings screen. I ended up putting this in spot 2, left column, and moved the Feeds settings to top of the middle column. But I don't stare at this screen much, so it's a guess.